### ISSUE TITLE
Get model working with new data

### ISSUE THREAD

**Issue by JackKelly** [openclimatefix/power_perceiver#88]
# TODO to get model working
- [x] Set UNet to 256x128
- [x] Set power perceiver to use 64x64 crop
- [x] Continue on from line 523 in experiment 024
- [x] Handle examples where GSP and PV are both NaN. & examples where just PV is NaN.
    - [x] Mask attention queries into Satellite Transformer
    - [x] Mask attention queries into FullModel.time_transformer
    - [x] Mask loss
- [x] Subselect the timesteps within the model, during training (mimic `xr_batch_processor.reduce_num_timesteps`, because we can't use that, because we don't have `xr_batch_processors` in raw, because it's hard to stack DataArrays with different PV system IDs.) But always use the full set of satellite images to train the SatellitePredictor.
- [x] Increase batch size (after subselecting timesteps)
- [x] change paths
- [x] #89
- [x] Move the SatellitePredictor training code into `FullModel`
- [x] Remove GSP timestamp from GSP query generator, when used for satellite transformer
- [x] Use multiple `num_workers`. Remove `raw_dataset.per_worker_init`. Try with the hack that allows fsspec to be forked. `fsspec` no longer has an `asyn` module. Need to read the `fsspec` github issues.
- [x] increase `min_duration_to_load_per_epoch`
